atlite/gis.py

```diff
@@ -433,12 +433,12 @@ def shape_availability(geometry, excluder):
 
     bounds = rio.features.bounds(geometry)
     transform, shape = padded_transform_and_shape(bounds, res=excluder.res)
-    masked = geometry_mask(geometry, shape, transform).astype(int)
-    exclusions.append(masked)
+    masked = geometry_mask(geometry, shape, transform)
+    exclusions = masked
 
     # For the following: 0 is eligible, 1 in excluded
     raster = None
-    for d in excluder.rasters:
+    for idx, d in enumerate(excluder.rasters, start=1):
```

Comment:
why using `enumerate` here?

Reply:
Enumerate not necessary here, you are right. An artefact from merging the inital local changes. Removed it.

Reply:
`enumerate` not necessary here, you are right. An artefact from merging the inital changes locally. Removed it.